stats.rst

```diff
@@ -0,0 +1,209 @@
+
+.. code-block:: console
+
+  $ sudo /usr/lib/python3/dist-packages/django/bin/django-admin.py migrate --settings=graphite.settings
+
+
+And you'll have to graphite (``_graphite``) access to some necessary files.
+
+.. code-block:: console
+
+  $ sudo chown -R _graphite:_graphite /var/lib/graphite/
+  $ sudo chown -R _graphite:_graphite /var/log/graphite/
+
+^^^^^^^^^^^^^^^^^^
+Installing Apache2
+^^^^^^^^^^^^^^^^^^
+
+The next thing you need to run Graphite's Web app is a web server. Here I'll show you how to set up ``Apache2``.
+
+.. code-block:: console
+
+  $ sudo apt-get install -y apache2 libapache2-mod-wsgi-py3
+  $ sudo cp /usr/share/graphite-web/apache2-graphite.conf /etc/apache2/sites-available
+  $ sudo a2dissite 000-default
+  Site 000-default disabled.
+  $ sudo a2ensite apache2-graphite
+  Enabling site apache2-graphite.
+  $ sudo systemctl reload apache2
+
+If everything went well you should be able to see the graphite dashboard by going to ``127.0.0.1`` or ``0.0.0.0`` in your web browser
+
+--------------------------------
+Running Stats Collection in Flux
+--------------------------------
+
+Once you have brubeck and Graphite setup and Graphite running there are only two more things left to be able to get stats from ``flux``. 
+First you will need to set the ``FLUX_FRIPP_STATSD`` environment variable with the endpoint of the aggregator (brubeck in this case).
+If you just used the default config for brubeck as suggested earlier, the endpoint is ``0.0.0.0:8126``. 
+
+.. code-block:: console
+
+  $ export FLUX_FRIPP_STATSD=hostname:port
+
+The second thing needed is to run brubeck. There are a few option, but no real best way as of now.
+
+.. code-block:: console
+
+  # launch brubeck as a background job in flux
+  $ flux start
+  $ flux mini run ~/path/to/brubeck --config=~/path/to/config.default.json
+
+  # run brubeck in a seperate terminal
+  $ cd brubeck
+  $ ./brubeck
+
+  # run bubeck in the background outside of flux
+  $ cd brubeck
+  $ ./brubeck &
+  $ flux start
+
+  # launch brubeck as a background job in flux on flux start
+  $ echo 'flux mini run ~/path/to/brubeck --config=~/path/to/config.default.json' >> etc/rc1
+  $ flux start
+
+-------------------------------
+Configuring Graphite (Optional)
+-------------------------------
+
+By default Graphite will plot points once per minute. For use within flux, one minute is a long time and often causes some loss of information.
+To change the interval at which Graphite (really carbon) is going to be plotting data there are a few things you need to do.
+
+First make sure that the carbon cache is enable on boot.
+
+.. code-block:: console
+
+  $ sudo vim /etc/default/graphite-carbon
+  # set to true
+  CARBON_CACHE_ENABLED=true
+
+Next, you will need to edit carbon's config file and set the specific you want. You can fin specifics on the configurations `here <https://graphite.readthedocs.io/en/stable/config-carbon.html>`_.
```

Comment:
Some typos in this line "and set the specific you want" and "You can fin specifics on the configurations"